netifi-broker-client/src/main/java/com/netifi/broker/DefaultBrokerService.java

```diff
@@ -440,52 +435,52 @@ private WeightedReconnectingRSocket createWeightedReconnectingRSocket() {
         higherQuantile,
         inactivityFactor);
   }
-
-  @Override
-  public BrokerSocket group(CharSequence group, Tags tags) {
-    return new DefaultBrokerSocket(
-        payload -> {
-          ByteBuf data = payload.sliceData().retain();
-          ByteBuf metadataToWrap = payload.sliceMetadata();
-          ByteBuf metadata =
-              GroupFlyweight.encode(ByteBufAllocator.DEFAULT, group, metadataToWrap, tags);
-          Payload wrappedPayload = ByteBufPayload.create(data, metadata);
-          payload.release();
-          return wrappedPayload;
-        },
-        this::selectRSocket);
-  }
-
-  @Override
-  public BrokerSocket broadcast(CharSequence group, Tags tags) {
-    return new DefaultBrokerSocket(
-        payload -> {
-          ByteBuf data = payload.sliceData().retain();
-          ByteBuf metadataToWrap = payload.sliceMetadata();
-          ByteBuf metadata =
-              BroadcastFlyweight.encode(ByteBufAllocator.DEFAULT, group, metadataToWrap, tags);
-          Payload wrappedPayload = ByteBufPayload.create(data, metadata);
-          payload.release();
-          return wrappedPayload;
-        },
-        this::selectRSocket);
-  }
-
-  @Override
-  public BrokerSocket shard(CharSequence group, ByteBuf shardKey, Tags tags) {
-    return new DefaultBrokerSocket(
-        payload -> {
-          ByteBuf data = payload.sliceData().retain();
-          ByteBuf metadataToWrap = payload.sliceMetadata();
-          ByteBuf metadata =
-              ShardFlyweight.encode(
-                  ByteBufAllocator.DEFAULT, group, metadataToWrap, shardKey, tags);
-          Payload wrappedPayload = ByteBufPayload.create(data, metadata);
-          payload.release();
-          return wrappedPayload;
-        },
-        this::selectRSocket);
-  }
+  //
+  //  @Override
+  //  public BrokerSocket group(CharSequence group, Tags tags) {
```

Comment:
why is this commented out? is it dead code now?

Reply:
Yes :) I ported this stuff from the work in proteus-java and that was commented there and moved into the BrokerClient methods